### PR TITLE
fix(deps): run Snyk wizard

### PR DESCRIPTION
**Pull request by mantariksh**
This PR upgrades the following packages to resolve vulnerabilities identified by Snyk:
- `file-loader`
- `babel-loader`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -201,7 +201,7 @@
     "@typescript-eslint/parser": "^4.9.0",
     "auto-changelog": "^2.2.1",
     "axios-mock-adapter": "^1.19.0",
-    "babel-loader": "^8.2.2",
+    "babel-loader": "^8.2.1",
```
Comment (liangyuanruo) [inline package.json:204]: snyk wizard recommended a version downgrade?

Comment (liangyuanruo) [reply package.json:204]: package [seems ok](https://snyk.io/test/npm/babel-loader/8.2.2)

File: package.json [hunk 2]
```diff
@@ -115,7 +115,7 @@
     "express-session": "^1.15.6",
     "express-winston": "^4.0.5",
     "fetch-readablestream": "^0.2.0",
-    "file-loader": "^4.0.0",
+    "file-loader": "^4.3.0",
```
Comment (liangyuanruo) [inline package.json:118]: funnily enough, file-loader [seems fine too](https://snyk.io/test/npm/file-loader/4.0.0)

**Review by liangyuanruo — CHANGES_REQUESTED** [review]
to avoid downgrading babel-loader

File: package.json (hunk 1 above)
Comment (mantariksh) [reply package.json:204]: snyk works in mysterious ways

**Review by liangyuanruo — APPROVED** [review]

**Pull request merged**